docs/modules/ROOT/pages/support/data-privacy.adoc

```diff
@@ -10,12 +10,6 @@ If you use a public GitHub repository as a component’s source, {ProductName} p
 
 *Note:* There is no functionality to use a private Quay repository. The artifacts {ProductName} builds are public and visible to anyone who is viewing the Quay repository. 
```

Comment:
You can use a private Quay repository now, and we need docs for how to do that.  See Slack.

Reply:
FYI
To be able to use a private Quay.io repo, you'd need to import a private Git Repo, which is being addressed in https://github.com/redhat-appstudio/docs.appstudio.io/pull/173 . 

Is that what you implied, @amfred?